website/cron.py

```diff
@@ -23,36 +23,36 @@ def unanswered_notification(self):
         import datetime as DT
 
         try:
-            weekago = DT.date.today() - DT.timedelta(days = 6)
+            weekago = DT.date.today() - DT.timedelta(days=6)
             questions = Question.objects\
-                        .filter(date_created__lte = weekago, is_spam = 0)
+                .filter(date_created__lte=weekago, is_spam=0)
         except Exception as e:
```

Comment:
Just a suggestion below:

What Exception is caught?
Also, filter is used so if no match then it will return empty set. But in exception it is "No questions found".
Also, print can be removed.

Reply:
This is the cron file. Its independent code and the changes shown are just pep8 fixes. We will figure out later.